themes-default/slim/views/manage_failedDownloads.mako

```diff
@@ -21,7 +21,8 @@ const startVue = () => {
         },
         data() {
             return {
-                header: 'Failed Downloads'
+                header: 'Failed Downloads',
+                limit: '${limit}'
```

Comment:
Just note that here you do it differently than on `history.mako` (`limit: parseInt('${limit}', 10)`)